commission/reports/report_settlement_templates.xml

```diff
@@ -51,21 +51,31 @@
                                 </td>
                             </tr>
                         </tbody>
-                        <tfoot>
-                            <tr>
-                                <td />
-                                <td class="text-end">
-                                    <strong>Total</strong>
-                                </td>
-                                <td class="text-end">
-                                    <span
-                                        t-field="o.total"
-                                        t-options="{'widget': 'monetary', 'display_currency': o.currency_id}"
-                                    />
-                                </td>
-                            </tr>
-                        </tfoot>
                     </table>
+                    <div class="clearfix mb-4">
```

Comment:
Why are you putting `clearfix`? That kind of dirty patches amending the whole thing should be avoided.